### ISSUE TITLE
Manager Date Search

### ISSUE THREAD

**Issue by jaypeasee** [jaypeasee/overlook-hotel#30]
### Create functionality for the Manager to view available rooms for any date.
When these room cards appear, there should be a button and input for the Manager to be able to book for a Guest.
